### DISCUSSION TITLE
Root path "/home/$USER/SquishBox" not found!

### DISCUSSION THREAD

**Discussion by hasis** [GeekFunkLabs/fluidpatcher#99] [General]
Setup: Raspberry Pi 4, RaspOS Lite (2023-10-10)

Fresh install of the script, pressed [Y] for Web server, but web server returning a `Root path not found` error (technically this is a 200).

Running the `headlesspi.py` script directly (or via bashrc) seems to work:

```
nikita@raspberrypi:~ $ ./headlesspi.py 
fluidsynth: warning: Requested a period size of 64, got 444 instead
fluidsynth: Connection of Midi Through Port-0 succeeded
Loading bank 'bank1.yaml' .. 
fluidsynth: warning: Sample 'SineWave': ROM sample ignored
Bank loaded.
Selected patch 1/26: Piano
```

But no luck or obvious errors with the WebUI.

Thanks for the project, 🤞I can get this bit working! :)

**Comment by albedozero** [comment]
This seems to be another change in Raspberry Pi OS moving from bullseye (debian 11) to bookworm (debian 12): home directories have permission 700 by default, which means the web server cannot view the contents of SquishBox/ in your home directory. The installer has been updated to fix this - if you re-run the installer and just install the web server it should fix things. You could also just enter the command:

```bash
sudo chmod g+rw /home/pi
```

replacing `pi` with your username, assuming you installed fluidpatcher in your home directory.

**Reply by hasis** [reply]
Very cool, thank you!
